site/source/tutorials/accessibility.md

```diff
@@ -0,0 +1,96 @@
+
+## Using Dojo 2 widgets
+
+All widgets provided through the `@dojo/widgets` package have been created with a range of assistive technology and perceptual differences in mind. However, they are not foolproof. There are a number of instances where additional information is required before peak accessibility can be achieved. Often, a few properties are required to enable a widget is to be fully accessible by all users. To help developers achieve this goal, each widget’s example page models best practices. Additionally, many of the properties exposed by a widget are related to accessibility, providing valid values for those properties allows a developer to be sure that they are not missing any a11y enhancements.
+
+### Form controls
+
+All simple form controls, such as `TextInput`, `Textarea`, and `Select`, use either a styled native control or provide a native fallback, which allows them to take advantage of built-in accessibility features wherever possible. Properties are provided in the widget interface to control relevant ARIA (Accessible Rich Internet Applications Suite) attributes like `aria-invalid`, `aria-describedby`, and `aria-readonly`.
+
+For example, a `TextInput` widget with the following properties:
+
+```ts
+w(TextInput, {
+    describedBy: ‘foo’,
+    disabled: true,
+    invalid: true,
+    type: ‘email’
+});
+```
+
+Would create the following attributes on the input node:
+`<input type=“email” aria-describedby=“foo” aria-invalid=“true” disabled>`
+
+### Label
+
+Most form widgets contain a `label` property that allows users to easily associate a text label with the control along with options to position it or keep it visually hidden. `Label` is also available as a separate widget for use with custom form controls.
+
+The `label` property on both form widgets and the `Label` widget accepts either a string or an [options object](https://github.com/dojo/widgets/blob/master/src/label/Label.ts#L13-17) that allows customization of where the label is placed (before or after the input), and whether it is visually hidden. Visually hiding a label is done by using screen reader-accessible CSS styles to hide text content, and is recommended for use with form controls that have no visible label.
```

Comment:
visually hidden. Visually hiding  (a bit redundant)...  perhaps "visually hidden. Screen reader-accessible CSS rules are use to visually hide text content, and is recommended for use with form controls with no visible label"